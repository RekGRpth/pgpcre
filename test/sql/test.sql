SELECT pcre_text_eq('foo', 'fo+');
SELECT pcre_text_eq('bar', 'fo+');
SELECT pcre_text_eq('error', '+');

SELECT 'foo' =~ 'fo+';
SELECT 'bar' =~ 'fo+';
SELECT 'error' =~ '+';
