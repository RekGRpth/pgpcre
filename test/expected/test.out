SELECT pcre_text_eq('foo', 'fo+');
 pcre_text_eq 
--------------
 t
(1 row)

SELECT pcre_text_eq('bar', 'fo+');
 pcre_text_eq 
--------------
 f
(1 row)

SELECT pcre_text_eq('error', '+');
ERROR:  PCRE compile error: nothing to repeat
